aii-core/src/main/perl/Shellfe.pm

```diff
@@ -935,9 +969,16 @@ sub cmds
             foreach my $node (@reinstall_list) {
                 $nodes{$node}->{reinstall} = 1 if exists($nodes{$node});
             }
-
-            $self->$cmd (%nodes);
+            # If needed, do DHCP here
+            if ("$cmd" eq "configure" && !$self->option(NODHCP)){ 
+                $self->change_dhcp(CONFIGURE, %nodes);
+            };
             $self->info (scalar (keys (%nodes)) . " nodes to $cmd");
+            $self->$cmd (%nodes);
+            if ("$cmd" eq "remove" &&  !$self->option(NODHCP)) {
```

Comment:
I didn't want to change the order. It was implemented before in configure: aii -> plugins; remove: plugins -> aii. Don't know if this is by design / needed for the plugins